cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -44,6 +44,15 @@ extern "C" {
 #endif
 
 /**
+ * @name    Power management configuration
+ * @{
+ */
+#if defined(CPU_FAM_STM32F1) || defined(CPU_FAM_STM32F2) || defined(CPU_FAM_STM32F4)
+#define PROVIDES_PM_LAYERED_OFF
```

Comment:
Should be `#define PROVIDES_PM_OFF` in my opinion. As the function is still called `pm_off()`.

Reply:
nope, as we want to explicitly override the `pm_off` function provided by the `pm_layered` module.

Rationale: for the 3 CPU families specified above, we know that we use the `pm_layered` module. So if we want to use a custom `pm_off` function, we need to explicitly override the one provided by `pm_layered` (which in turn overrides the default one from `periph_common`).